### ISSUE TITLE
Dino Dave has many voices talking around him which causes lag and save file memory leaks.

### ISSUE THREAD

**Issue by JosephTheAmazing** [CleverRaven/Cataclysm-DDA#76670]
### Describe the bug

When I am near the entrance of the refugee camp building, the game starts lagging. I have found that it seems that the cause of the lag is a huge amount of talking happening near Dino Dave. The glitch happens as long as the player is near enough to him to have the game calculate conversations near him. Another effect of the bug is that when you save while the glitch is happening, the save file will start to increase in size rapidly. Other then the talking I am not sure what exactly is causing this bug. I am playing vanilla version 2024-09-22-1507.

### Attach save file

This is the save file.
[Glitch save file.zip](https://github.com/user-attachments/files/17142124/Glitch.save.file.zip)

### Steps to reproduce

1. Go to the refugee center in the attached save file.
2. Go to Dino Dave.
3. Once at Dino Dave, the game will be lagging and many conversations will be happening for no reason near Dino Dave.

### Expected behavior

The game not lagging and the save file not increasing in size for no reason if saved during the glitch.

### Versions and configuration

OS: Windows  10
Game version: 2024-09-22-1507
Graphics version: Tiles and sounds
Ingame language: English

**Comment by BaumgeistOne** [comment]
I have noticed a big performance drop around the entrance of the refugee center too.

**Comment by harakka** [comment]
Yup, `o.1.-1` is 70MB in size and `weapon_talk_hallucination` occurs in it 1088050 times. Probably not normal.

**Comment by Rowmaster3** [comment]
Did a bit of digging. Seems to be related to his "Kaluptic Psychosis" Mutation. As turning it off causes him to stop. Turning it back on causes it to start again after a day or 2. If you want a quick fix, go to the "debug menu" - "Player" - "Edit player/NPC" - "Dino Dave" - "Mutate" - Turn off Kaluptic Psychosis. Takes a few turns to clear the backlog of him talking.

**Comment by JosephTheAmazing** [comment]
Thanks
